### PR TITLE
Sprint12 04 repo lwrps maual review fixes

### PR DESCRIPTION
**Pull request by Neve**
- Indentation fix
- log style fix (brackets reomoved)
- More comments 
- Removed "unless node[:platform] == mac_os_x " from default recipes
- a45414 repo_git does not return a notice if git calls fail ( https://wush.net/trac/rightscale/ticket/4581 )

### CODE REVIEW

File: cookbooks/repo_git/recipes/default.rb [hunk 1]
```diff
@@ -8,7 +7,6 @@
 
 rightscale_marker :begin
 
-unless node[:platform] == "mac_os_x" then
   # Install git client
```
Comment (douglaswth) [inline cookbooks/repo_git/recipes/default.rb]: If you are going to remove a conditional, you should fix the indentation of the surrounded code.

File: cookbooks/repo_svn/recipes/default.rb [hunk 2]
```diff
@@ -7,9 +7,6 @@
 
 rightscale_marker :begin
 
-PROVIDER_NAME = "repo_svn"
-
-unless node[:platform] == "mac_os_x" then
   # install subversion client
```
Comment (douglaswth) [inline cookbooks/repo_svn/recipes/default.rb]: Again, if you are going to remove a conditional, you should fix the indentation of the surrounded code.

File: cookbooks/repo_git/recipes/default.rb (hunk 1 above)
Comment (Neve) [inline cookbooks/repo_git/recipes/default.rb]: changes done

File: cookbooks/repo_git/providers/default.rb [hunk 3]
```diff
@@ -27,27 +27,27 @@
       branch = new_resource.revision
 
       Dir.chdir new_resource.destination
-      puts "Updating existing git repo at #{new_resource.destination}"
-      puts `git pull`
+      Chef::Log.info "  Updating existing git repo at #{new_resource.destination}"
+      Chef::Log.info `git pull`
```
Comment (ekargard) [inline cookbooks/repo_git/providers/default.rb]: Should the command be executed and the output captured and logged.  It seems a little strange to have a log command execute a command.

Comment (ekargard) [inline cookbooks/repo_git/providers/default.rb]: Also - the return code is ignored.  Does Chef::Log handle errors in the same manner as puts ``?

Comment (douglaswth) [inline cookbooks/repo_git/providers/default.rb]: Backticks just return the string of the output of whatever command. There is no error handling either way, both puts and Chef::Log will receive the output string and output according to that.

Comment (Neve) [inline cookbooks/repo_git/providers/default.rb]: To get error handling 
we can convert ruby_blocks to bash_script
or we can use system() - but we will loose output in this case

Comment (douglaswth) [inline cookbooks/repo_git/providers/default.rb]: I think the output of bash blocks is logged.

Comment (Neve) [inline cookbooks/repo_git/providers/default.rb]: repo_git and repo_svn :pull actions refactored

File: cookbooks/repo_ros/providers/default.rb [hunk 4]
```diff
@@ -67,34 +68,39 @@
...
-   #moving dir with downloaded and unpacked ROS source to temp folder
-   #to prepare source for capistrano actions
-   bash "Moving #{new_resource.destination} to #{repo_dir}/ros_repo/" do
+  #The embedded chef capistrano resource can work only with git or svn repositories
+  #  After code download from ROS storage, we transform this code repo to git
+  #  Then we apply capistrano chef provider
+  #  After that we remove all git information from new repo (.git folders)
+
+  # Moving dir with downloaded and unpacked ROS source to temp folder
+  #   to prepare source for capistrano actions
```
Comment (douglaswth) [inline cookbooks/repo_ros/providers/default.rb]: The text in comments should be indented one space from the #.

**Pull request merged**